README.md

```diff
@@ -1,21 +1,44 @@
 Cypher Editor
 =============
 
-The goal of this project is to provide feature-rich support
-of [Cypher](http://www.opencypher.org/) query language in
-JavaScript-based editors. 
+<p align="center">Cypher-editor provide feature-rich support of Cypher query language in JavaScript-based editors</p>
 
-It includes, but not limited to functionality such as:
+<p align="center"><img src="https://cloud.githubusercontent.com/assets/24562497/25952124/6942acb6-3668-11e7-8323-05802debdefc.png" width=700 alt="Screenshot of Example Documentation created with Slate"></p>
 
-* Syntax highlight
-* Error reporting
-* Autocompletion
+<p align="center"><em>The example above is using cypher-editor. Check it out at <a href="https://github.com/neo4j/neo4j-browser">neo4j/neo4j-browser</a>.</em></p>
+
+### Key Features
+
+* Syntax highlighting
+* Auto-complete for labels, relationship types, properties and variables
+* Command auto-complete
 
 ### Supported editors
 
 * [CodeMirror](cypher-codemirror/)
+* [Cypher-editor](cypher-editor-support)
```

Comment:
This is NOT a supported editor.

In previous version "Cypher Editor Support" module had it's own sections for a reason. Because it's a "backend" module, that provide different capabilities, but do not represent any editor integration.